### ISSUE TITLE
Support for ModelForm

### ISSUE THREAD

**Issue by ailling** [scholrly/neo4django#135]
ModelForm doesn't seem to be supported. I have the following class:

```
from neo4django.db import models

class Person(models.NodeModel):
    name = models.StringProperty()
```

And the following modelform:

```
class PersonForm(forms.ModelForm):
    class Meta:
        model = Person
```

The generated error is:
`'super' object has no attribute 'editable'`

When django dynamically resolves fields it's finding a BoundProperty instead of a StringProperty or Property for this field. This happens around line 148 in django/forms/models.py
Example pdb output:

```
146         opts = model._meta
147         import pdb
148         pdb.set_trace()
149         for f in sorted(opts.fields + opts.many_to_many):
150  ->         if not f.editable:
151                 continue
152             if fields is not None and not f.name in fields:
153                 continue
154             if exclude and f.name in exclude:
155                 continue
(Pdb) opts.fields
[<django.db.models.fields.AutoField: id>, <neo4django.db.models.properties.BoundProperty object at 0x7f90a0532290>]
```

editable is defined for a Property, but not a BoundProperty and this line will fail.

Any idas on how to fix this, or am I consuming this incorrectly?

**Comment by mhluongo** [comment]
Sorry I took so long to respond! I read over this issue and the associated StackOverflow question a couple days ago, but don't yet have a better fix over the workaround you're using.

I do, however, have a good idea how to fix the problem. `BoundProperty` proxies many of it's attributes back to a `Property` instance. `editable` should be in that list of proxied properties. I'm not sure whether that will fully enable the use of `ModelForm`s, however.

I'm pretty busy right now, and since we don't use `ModelForm`s currently this isn't as high a priority as it should be. This would be an awesome opportunity to contribute- would you be interested? If not a fix, a few tests confirming the issue and asserting the proper behavior would be really helpful.

**Comment by ailling** [comment]
Thanks for looking into this- the solution I posted on stack overflow is suitable for my needs & personally, I don't find ModelForms particularly useful in the long-term anyway because I tend to replace them with actual form classes so I can have more flexibility (i.e., custom validation errors, initialization routines, etc.)

My initial thought was also to add an editable field to the BoundProperty since it looks like that field didn't proxy over properly, which worked but general ModelForm support would require neo4django shipping with definitions of fields that map to the properties, so there would still be a lot of work going into what seems to be a small gain at the end of the day.

For now I think the workaorund I suggested will suffice for 95% of cases, and works me for :)

I'm making heavy use of your library & will likely end up contributing at some point, but I need to learn more about the library first.

Thanks for the reply! I'm closing this issue.
